### DISCUSSION TITLE
Need test.pypi.org index for python package in requirements.txt

### DISCUSSION THREAD

**Discussion by PurplefinNeptuna** [Comfy-Org/ComfyUI-Manager#918] [Q&A]
I currently contribute to [crystian/ComfyUI-Crystools](https://github.com/crystian/ComfyUI-Crystools) for bringing monitoring for AMD gpus
But it has problems, mainly `pyrsmi` package in pypi.org index is extremely outdated, only updated in test.pypi.org index as written in [ROCm/pyrsmi](https://github.com/ROCm/pyrsmi) readme.
was tried to use `-e` in requirements.txt but it throw errors for manager user.
how to do it rightly without `importlib.util.find_spec()` and telling amd users to install package manually?

**Comment by ltdrdata** [comment]
Just patched.
From ComfyUI-Manager 2.48.4 onwards, you can provide the --index-url specification in requirements.txt as follows:

```
pyrsmi --index-url https://test.pypi.org/simple
```

**Reply by PurplefinNeptuna** [reply]
can I use strict version? it seems it cant because:
https://github.com/ltdrdata/ComfyUI-Manager/blob/55cb4c3d0ae00bb03c3e8550cc4f586b4d6cf738/prestartup_script.py#L451
group 3 from `pyrsmi==0.0.6 --index-url https://test.pypi.org/simple` will be `0.0.6 --index-url https://test.pypi.org/simple`

**Reply by ltdrdata** [reply]
This is fixed.
